### ISSUE TITLE
qt.py example has display issues

### ISSUE THREAD

**Issue by feenes** [cztomczak/cefpython#630]
I'd like to a cefpython3 window into a qt application and in order to get started I tried the qt.py example.
( https://github.com/cztomczak/cefpython/blob/master/examples/qt.py )
(  Latest commit 181588c on Jan 22, 2020  )

The qt.py example doesn't seem to work properly.

- The application starts
- a window with the title pqtXXX example shows up
- adress bar shows up with three gray buttons and https://www.google.com as url location
- the contents if google (request to confirm / modify cookie settings shows up

This all seems good, but then.

- the same contents shows up a second time in the window, but this time with reduced size
- using the scroll bar the 'small contents can be scrolled away'
- as soon as I mouse_over a button the 'small windows shows up again'

tested with 
- windows python3.6-64 and pyqt4 
- windows python 3.6-64 and pyqt5
- windows python 2.7-64 and pyqt4
- windows python 2.7-64 and pyside

Is this issue known? Is there a fix?

**Comment by cztomczak** [comment]
Are you having DPI issues? See this comment: https://github.com/cztomczak/cefpython/issues/530#issuecomment-505066492

**Comment by feenes** [comment]
Yes indeed. this is the issue.
Thank you very much

I will try embedding params in the manifest llateron

For the time being I just set scaling to 100% and the problem disappeared.

I wonder whether there is any way to make it easier to understand that issue when it is encountered.

Would there be any way, that the sample code detects that issue?

Or if not perhaps just add a print statement to `qt.py`, that says to look at https://github.com/cztomczak/cefpython/issues/530#issuecomment-505066492 if there's anything strange with the display sizes?

I guess I'm not the only one encountering this issue and being quite confused and anything, that helps to pinpoint the issue faster could be great

**Comment by cztomczak** [comment]
Yeah, we could add some comments in code and add info to KnowledgeBase document as well.

**Comment by feenes** [comment]
What would be best, but I'm not sure this is technically feasible, would be to detect the inconsistency between dpi and window size and report it. 

Thanks a again. 

Shall I close the issue?

**Comment by cztomczak** [comment]
I don't think you can detect issue programmatically.
